CMakeLists.txt

```
@@ -61,6 +63,19 @@ add_executable(kcd2-386 EXCLUDE_FROM_ALL)
 target_sources(kcd2-386 PRIVATE "${PROJECT_SOURCE_DIR}/kauai/src/kcd2_386.c")
 target_include_directories(kcd2-386 PRIVATE $<TARGET_PROPERTY:kauai,INCLUDE_DIRECTORIES>)
 
+if ("${CMAKE_GENERATOR_PLATFORM}" STREQUAL "Win32")
+  set(AUDIOMAN_LIB 3DMMForever::AudioMan)
+else()
+  add_library(audioman)
+  target_include_directories(audioman PUBLIC
+      "${PROJECT_SOURCE_DIR}/kauai/src"
+  )
+  target_sources(audioman PRIVATE
+      "${PROJECT_SOURCE_DIR}/audioman/audioman.cpp"
+  )
+  set(AUDIOMAN_LIB audioman)
+endif()
```

```suggestion
if (NOT TARGET 3DMMForever::AudioMan)
  add_library(audioman)
  add_library(3DMMForever::AudioMan ALIAS audioman)
  target_include_directories(audioman PUBLIC "${PROJECT_SOURCE_DIR}/kauai/src")
  target_sources(audioman PRIVATE "${PROJECT_SOURCE_DIR}/audioman/audioman.cpp")
endif()
```

Creating an alias when the found package does not exist might be a better option here.